src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -819,8 +825,10 @@ impl AstDisplay for ConnectionOptionName {
             ConnectionOptionName::SslKey => "SSL KEY",
             ConnectionOptionName::SslMode => "SSL MODE",
             ConnectionOptionName::SessionToken => "SESSION TOKEN",
+            ConnectionOptionName::CatalogType => "TYPE",
```

Comment:
This is the only option whose string representation does not match its enum name. Is there a good reason this is not `"CATALOG TYPE"`?